content/SCALE/SCALEUIReference/SystemSettings/EnclosureScreensSCALE.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "View Enclosure Screen"
+description: "This article provides information on the TrueNAS **View Enclosure** screen, the information you can find there."
+weight: 60
+aliases:
+tags:
+ - scaleenclosure
+ - scalepools
+ - scaledisks
+ - scaledashboard
+---
+
+{{< toc >}}
+The **View Enclosure** screen displays an image of the TrueNAS-provided system hardware with drive images you can select. 
+The screen includes information on system pools, disks and their status, details and the HDD stats change with the drive you select on the system image.
```

Comment:
```
Simplify?

The screen includes information on system pools, disks and their status, details and the HDD stats change with the drive you select on the system image.
```

Reply:
I'm going to reword this section as I'm not a fan of what I wrote.